core/src/main/resources/jenkins/agents/Messages_cs.properties

```diff
@@ -0,0 +1 @@
+IOfflineCause.offline=nedostupnÃ½
```

Comment:
I think you have some mojibake issues? Compare https://github.com/jenkinsci/jenkins/blob/3983ac69e2bf8972e1294e7c57714892e898cadf/core/src/main/resources/jenkins/agents/Messages_cs.properties#L1 to what I guess is the original, https://github.com/jenkinsci/jenkins/blob/3983ac69e2bf8972e1294e7c57714892e898cadf/core/src/main/resources/lib/hudson/executors_cs.properties#L29

Reply:
Doublechecking...